demo/components/page-transition/PageTransition.module.css

```diff
@@ -0,0 +1,118 @@
+.fade {
+    will-change: opacity;
+    transition: opacity 0.6s;
+
+    &.enter {
+        opacity: 0;
+    }
+
+    &.enterActive,
+    &.enterDone {
+        opacity: 1;
+    }
+
+    &.exitActive,
+    &.exitDone {
+        opacity: 0;
+    }
+}
+
+.slideLeft {
+    transition: transform 0.6s;
+    transform-style: preserve-3d; /* Necessary to performance on Firefox */
```

Comment:
This is odd. You're not working with 3d, so why does firefox require this for performance?
Does this intentionally promote the element into a composite layer in firefox? If so, I would prefer if you just would use translate3d 😄

Reply:
Yep, this is indeed odd but necessary, otherwise Firefox would stutter in `glideLeft` and `glideRight` animations. This has to do with the position fixed elements going out of the parent container, which has `overflow-x: hidden`.

I've changed to `backface-visibility: hidden` which has the same effect and actually seems better in terms of performance.

Anyways, I will put a note on the README about caveats like this one.